typescript/audio-summarizer/README.md

```diff
@@ -0,0 +1,188 @@
+# Emojifier 🤖🎨 - A simple audio summarizer with observability on Literal AI
+
+In this cookbook, we will start with a simple audio summarizer using the Web Speech API and Next.js. We will then add observability to the application using Literal AI.
+
+The base application converts audio to text and summarize the text as a sequence of emojis. It consists of :
+
+- A web page with a simple UI to record audio, using the [AudioRecorder](https://www.npmjs.com/package/react-audio-voice-recorder) React component.
+- A Next.js API route to convert the audio to text using OpenAI's [Whisper API](https://openai.com/index/whisper/).
+- A Next.js API route to summarize the text using OpenAI's [GPT-4](https://openai.com/index/gpt-4/).
+
+You can test the base application by running the development server. The only pre-requisite is to have Node.js installed on your machine.
+
+```bash
+cd without-literal
```

Comment:
Ah, it wasn't obvious to me at first that there are 2 codebases (`with-literal` and `without-literal`) and that you are guiding the user to start with the `without-literal` codebase and how to add Literal AI features.

If so, maybe it's worth mentioning that. A user, at this stage of the reading may as well try the whole code base as a way to boostrap a Literal AI project to test Literal AI features, this starting with the the `with-literal` codebase.

Reply:
done, i've mentioned it in the "Getting started" section